obsei/preprocessor/text_cleaner.py

```diff
@@ -0,0 +1,108 @@
+import logging
+from typing import Any, Dict, List, Optional
+import string
+import re
+from unicodedata import normalize as unicode_decode
+
+import nltk
+from nltk.corpus import stopwords
+from nltk.stem import PorterStemmer
+from nltk.tokenize import word_tokenize
+
+from pydantic import PrivateAttr
+
+from obsei.analyzer.base_analyzer import AnalyzerRequest
+from obsei.preprocessor.base_text_cleaner import (
+    BaseTextCleaner,
+    BaseTextCleanerConfig,
+)
+
+logger = logging.getLogger(__name__)
+nltk.download('punkt')
+
+
+class TextCleaner(BaseTextCleaner):
+
+    def __init__(self, **data: Any):
+        super().__init__(**data)
+        self.stop_words: List = stopwords.words('english')
+        self.stemmer = PorterStemmer()
+        
+
+    def clean_input(
+        self,
+        input_list: List[AnalyzerRequest],
+        config: BaseTextCleanerConfig,
+        **kwargs
+    ) -> List[AnalyzerRequest]:
+        text_cleaning_functions: List = config.text_cleaning_functions or self.get_text_cleaning_functions()
+        for index, input in enumerate(input_list):
+            tokens: List[str] = self.tokenize_text(input.processed_text)
+            for text_cleaning_function in text_cleaning_functions:
+                print(tokens)
+                tokens = text_cleaning_function(tokens)
+            input_list[index] = " ".join(tokens)
```

Comment:
We need to update `input.processed_text` so I think it should be -
```
input.processed_text = " ".join(tokens)
```

Reply:
fixed